docker-compose.yml

```diff
@@ -8,7 +8,7 @@ services:
     command: npm run styleguide:ci -- --quiet
     hostname: ca-styleguide.test
     ports:
-      - 6006
+      - '6006:6006'
```

Comment:
I'm not convinced we should bind this port to localhost as users (developers) may already have the styleguide running and this would cause a clash. In this instance the docker container is only really here to support running tests, not to support development as we do not deploy design-system using docker.